<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<!-- saved from url=(0039)http://genome.ucsc.edu/cgi-bin/hgTables -->
<html><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><style type="text/css">@charset "UTF-8";[ng\:cloak],[ng-cloak],[data-ng-cloak],[x-ng-cloak],.ng-cloak,.x-ng-cloak,.ng-hide:not(.ng-hide-animate){display:none !important;}ng\:form{display:block;}.ng-animate-shim{visibility:hidden;}.ng-anchor{position:absolute;}</style><script>window.readcubeExtension = {
  version: '1.35',
  id: 'mfacblegickmnfpaebakfnlbfhpoegin'
};</script>

<meta http-equiv="Content-Security-Policy" content="default-src *; script-src &#39;self&#39; blob: &#39;unsafe-inline&#39; &#39;nonce-7XrxAeT7uqKhKVCaESUu8jIIOF00&#39; code.jquery.com www.google-analytics.com www.samsarin.com/project/dagre-d3/latest/dagre-d3.js cdnjs.cloudflare.com/ajax/libs/d3/3.4.4/d3.min.js cdnjs.cloudflare.com/ajax/libs/jquery/1.12.1/jquery.min.js cdnjs.cloudflare.com/ajax/libs/jstree/3.2.1/jstree.min.js cdnjs.cloudflare.com/ajax/libs/bowser/1.6.1/bowser.min.js cdnjs.cloudflare.com/ajax/libs/jstree/3.3.4/jstree.min.js login.persona.org/include.js ajax.googleapis.com/ajax maxcdn.bootstrapcdn.com/bootstrap d3js.org/d3.v3.min.js cdn.datatables.net; style-src * &#39;unsafe-inline&#39;; font-src * data:; img-src * data:;">
	

<meta http-equiv="Content-Script-Type" content="text/javascript">
<meta http-equiv="Pragma" content="no-cache">
<meta http-equiv="Expires" content="-1">
	<title>Schema for snp150	</title>
    <link rel="stylesheet" href="./Schema for snp150_files/HGStyle.css" type="text/css">
<link type="text/css" rel="stylesheet" href="chrome-extension://pioclpoplcdbaefihamjohnefbikjilc/content.css"></head>
<body class="hgTables cgi">
<center><div id="warnBox" style="display:none;"><center><b id="warnHead"></b></center><ul id="warnList"></ul><center><button id="warnOK"></button></center></div></center>

<noscript><div class='noscript'><div class='noscript-inner'><p><b>JavaScript is disabled in your web browser</b></p><p>You must have JavaScript enabled in your web browser to use the Genome Browser</p></div></div></noscript>
<script async="" src="./Schema for snp150_files/analytics.js"></script><script type="text/javascript" src="./Schema for snp150_files/jquery.js"></script>
<script type="text/javascript" src="./Schema for snp150_files/jquery.plugins.js"></script>
<link rel="stylesheet" href="./Schema for snp150_files/nice_menu.css" type="text/css">
<div id="main-menu-whole">

		<div id="main-menu"><div id="block-nice_menus-1" class="clear-block block block-nice_menus">

	<div class="upcorner"></div>
	<div class="blockbg">
		<div class="blockcontent">
						<div class="content">

<ul class="nice-menu nice-menu-down sf-js-enabled" id="nice-menu-1">

<li class="first even" id="home"><a href="http://genome.ucsc.edu/index.html"><span id="home-link">Home</span></a></li> 

<li class="menuparent" id="tools1"><a href="http://genome.ucsc.edu/cgi-bin/hgGateway?hgsid=667608919_c94x7S5B0funel50l3ukR6d4tOh5">Genomes</a>
	<ul style="display: none; visibility: hidden;">
	<li class="first"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?hgsid=667608919_c94x7S5B0funel50l3ukR6d4tOh5&amp;db=hg38&amp;position=lastDbPos">Human GRCh38/hg38</a></li>
	<li><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?hgsid=667608919_c94x7S5B0funel50l3ukR6d4tOh5&amp;db=hg19&amp;position=lastDbPos">Human GRCh37/hg19</a></li>
	<li><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?hgsid=667608919_c94x7S5B0funel50l3ukR6d4tOh5&amp;db=mm10&amp;position=lastDbPos">Mouse GRCm38/mm10</a></li>
	<li><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?hgsid=667608919_c94x7S5B0funel50l3ukR6d4tOh5&amp;db=mm9&amp;position=lastDbPos">Mouse NCBI37/mm9</a></li>
	<li><a href="http://hgdownload.soe.ucsc.edu/hubs/mouseStrains/hubIndex.html">Mouse: 16 strains</a></li>
	<li class="last"><a href="http://genome.ucsc.edu/cgi-bin/hgGateway?hgsid=667608919_c94x7S5B0funel50l3ukR6d4tOh5">Other</a></li>
        </ul>
</li>

<li class="menuparent" id="tools2"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?hgsid=667608919_c94x7S5B0funel50l3ukR6d4tOh5">Genome Browser</a>
	<ul style="display: none; visibility: hidden;">
	<li class="first"><a id="configureMenuLink" href="http://genome.ucsc.edu/cgi-bin/hgTracks?hgsid=667608919_c94x7S5B0funel50l3ukR6d4tOh5&amp;hgTracksConfigPage=configure">Configure</a></li>
	<li><a id="trackSearchMenuLink" href="http://genome.ucsc.edu/cgi-bin/hgTracks?hgsid=667608919_c94x7S5B0funel50l3ukR6d4tOh5&amp;hgt_tSearch=track+search">Track Search</a></li>
	<li><a id="cartResetMenuLink" href="http://genome.ucsc.edu/cgi-bin/cartReset">Reset All User Settings</a></li>
        </ul>
</li>


<li class="menuparent" id="tools3"><span>Tools</span>
	<ul style="display: none; visibility: hidden;">
	<li class="first"><a id="blatMenuLink" href="http://genome.ucsc.edu/cgi-bin/hgBlat?hgsid=667608919_c94x7S5B0funel50l3ukR6d4tOh5&amp;command=start">Blat</a></li>
	<li><a id="tableBrowserMenuLink" href="http://genome.ucsc.edu/cgi-bin/hgTables?hgsid=667608919_c94x7S5B0funel50l3ukR6d4tOh5">Table Browser</a></li>
	<li><a id="vaiMenuLink" href="http://genome.ucsc.edu/cgi-bin/hgVai?hgsid=667608919_c94x7S5B0funel50l3ukR6d4tOh5">Variant Annotation Integrator</a></li>
	<li><a id="diMenuLink" href="http://genome.ucsc.edu/cgi-bin/hgIntegrator?hgsid=667608919_c94x7S5B0funel50l3ukR6d4tOh5">Data Integrator</a></li>
	<li><a title="Explore gene/protein interactions and pathways" id="interactionMenuLink" href="http://genome.ucsc.edu/cgi-bin/hgGeneGraph?hgsid=667608919_c94x7S5B0funel50l3ukR6d4tOh5">Gene Interactions</a></li>
	<li><a id="gsMenuLink" href="http://genome.ucsc.edu/cgi-bin/hgNear?hgsid=667608919_c94x7S5B0funel50l3ukR6d4tOh5">Gene Sorter</a></li>
	<li><a id="ggMenuLink" href="http://genome.ucsc.edu/cgi-bin/hgGenome?hgsid=667608919_c94x7S5B0funel50l3ukR6d4tOh5">Genome Graphs</a></li>
	<li><a id="ispMenuLink" href="http://genome.ucsc.edu/cgi-bin/hgPcr?hgsid=667608919_c94x7S5B0funel50l3ukR6d4tOh5">In-Silico PCR</a></li>
	<li><a id="liftOverMenuLink" href="http://genome.ucsc.edu/cgi-bin/hgLiftOver?hgsid=667608919_c94x7S5B0funel50l3ukR6d4tOh5">LiftOver</a></li>
	<li><a id="visiGeneMenuLink" href="http://genome.ucsc.edu/cgi-bin/hgVisiGene?hgsid=667608919_c94x7S5B0funel50l3ukR6d4tOh5">VisiGene</a></li>
	<li class="last"><a href="http://genome.ucsc.edu/util.html">Other Utilities</a></li>
	</ul>
</li>
<li class="menuparent" id="mirrors"><span>Mirrors</span>
	<ul style="display: none; visibility: hidden;">
	<li class="first"><a href="http://genome.ucsc.edu/mirror.html">Third Party Mirrors</a></li>
	<li><a href="http://genome.ucsc.edu/goldenPath/help/mirror.html">Mirroring Instructions</a></li>
	<li id="geoMirrorMenu" class="noHighlight"><hr></li>
<li id="server1" class="noHighlight">
<img alt="X" width="16" height="16" style="float:left;" src="./Schema for snp150_files/greenChecksmCtr.png">
US Server</li>
<li id="server2">
<img alt="X" width="16" height="16" style="float:left;visibility:hidden;" src="./Schema for snp150_files/greenChecksmCtr.png">
<a href="http://genome-euro.ucsc.edu/cgi-bin/hgGateway?redirect=manual">European Server</a></li>
<li id="server3">
<img alt="X" width="16" height="16" style="float:left;visibility:hidden;" src="./Schema for snp150_files/greenChecksmCtr.png">
<a href="http://genome-asia.ucsc.edu/cgi-bin/hgGateway?redirect=manual">Asian Server</a></li>

	</ul>

</li>
<li class="menuparent" id="downloads"><span>Downloads</span>
	<ul style="display: none; visibility: hidden;">
	<li class="first"><a href="http://hgdownload.soe.ucsc.edu/downloads.html">Genome Data</a></li>
	<li><a href="http://hgdownload.soe.ucsc.edu/downloads.html#source_downloads">Source Code</a></li>
  <li><a href="https://genome-store.ucsc.edu/">Genome Browser Store</a></li>
	<li><a href="http://hgdownload.soe.ucsc.edu/downloads.html#utilities_downloads">Utilities</a></li>
	<li class="last"><a href="http://genome.ucsc.edu/goldenPath/help/ftp.html">FTP</a></li>
	<li class="last"><a href="http://genome.ucsc.edu/goldenPath/help/mysql.html">MySQL Access</a></li>
	</ul>
</li>
<li class="menuparent" id="myData"><span>My Data</span>
	<ul style="display: none; visibility: hidden;">
	<li><a id="sessionsMenuLink" href="http://genome.ucsc.edu/cgi-bin/hgSession?hgsid=667608919_c94x7S5B0funel50l3ukR6d4tOh5&amp;hgS_doMainPage=1">My Sessions</a></li>
	<li><a id="publicSessionsMenuLink" href="http://genome.ucsc.edu/cgi-bin/hgPublicSessions?hgsid=667608919_c94x7S5B0funel50l3ukR6d4tOh5">Public Sessions</a></li>
	<li><a id="trackHubsMenuLink" href="http://genome.ucsc.edu/cgi-bin/hgHubConnect?hgsid=667608919_c94x7S5B0funel50l3ukR6d4tOh5">Track Hubs</a></li>
	<li><a id="customTracksMenuLink" href="http://genome.ucsc.edu/cgi-bin/hgCustom?hgsid=667608919_c94x7S5B0funel50l3ukR6d4tOh5">Custom Tracks</a></li>
	<li class="last"><a id="customCompositeMenuLink" href="http://genome.ucsc.edu/cgi-bin/hgCollection?hgsid=667608919_c94x7S5B0funel50l3ukR6d4tOh5">Track Collection Builder</a></li>
	</ul>
</li>

<!-- OPTIONAL_VIEW_MENU -->

<li class="menuparent" id="help"><span>Help</span><ul style="display: none; visibility: hidden;">
<li><a href="http://genome.ucsc.edu/goldenPath/help/hgTablesHelp.html">Help on Table Browser</a></li>
<li><a href="http://genome.ucsc.edu/FAQ/">FAQs &amp; Search</a></li>
<li><a href="http://genome.ucsc.edu/goldenPath/help/hgTracksHelp.html">Browser Documentation</a></li>
<li><a href="http://genome.ucsc.edu/training/index.html">Training</a></li>
<li><a href="http://genome.ucsc.edu/contacts.html">Mailing Lists</a></li>
<li class="last"><a href="http://genomewiki.ucsc.edu/" target="_blank">GenomeWiki</a></li>
</ul>
</li>

<li class="menuparent last" id="aboutUs"><span>About Us</span>
	<ul style="display: none; visibility: hidden;">
	<li class="first odd"><a href="http://genome.ucsc.edu/goldenPath/newsarch.html">News</a></li>
	<li class="even"><a href="http://genome.ucsc.edu/goldenPath/pubs.html">Publications</a></li>
	<li class="even"><a href="http://genome.ucsc.edu/blog/">Blog</a></li>
	<li class="odd"><a href="http://genome.ucsc.edu/cite.html">Cite Us</a></li>
	<li class="even"><a href="http://genome.ucsc.edu/goldenPath/credits.html">Credits</a></li>
	<li class="odd"><a href="http://genome.ucsc.edu/goldenPath/releaseLog.html">Release Log</a></li>
	<li class="even"><a href="http://genome.ucsc.edu/staff.html">Staff</a></li>
	<li class="odd"><a href="http://genome.ucsc.edu/contacts.html">Contact Us</a></li>
	<li class="even"><a href="http://genome.ucsc.edu/conditions.html">Conditions of Use</a></li>
	<li class="odd"><a href="https://ucscgenomics.soe.ucsc.edu/careers/" target="_blank">Jobs</a></li>
	<li class="even last"><a href="http://genome.ucsc.edu/license/">Licenses</a></li>
</ul>
</li>

</ul>
</div>
		</div>
	</div>
	<div class="downcorner"></div>

</div>
</div><!-- end main-menu -->
</div><!-- end main-menu-whole -->

<a name="TOP"></a>
<table border="0" cellpadding="0" cellspacing="0" width="100%">

<!-- +++++++++++++++++++++ CONTENT TABLES +++++++++++++++++++ -->
<tbody><tr><td colspan="3">
<div id="firstSection">      <!--outer table is for border purposes-->
      <table width="100%" bgcolor="#888888" border="0" cellspacing="0" cellpadding="1"><tbody><tr><td>
    <table bgcolor="#FFFEE8" width="100%" border="0" cellspacing="0" cellpadding="0"><tbody><tr><td>
     <div class="subheadingBar"><div class="windowSize" id="sectTtl">
Schema for snp150     </div></div>
     <table bgcolor="#FFFEE8" width="100%" cellpadding="0"><tbody><tr><th height="10"></th></tr>
     <tr><td width="10">&nbsp;</td><td>


<b>Database:</b> hg38&nbsp;&nbsp;&nbsp;&nbsp;<b>Primary Table:</b> snp150&nbsp;&nbsp;&nbsp;&nbsp;<b>Row Count: </b>  335,322,261&nbsp;&nbsp;<b> Data last updated:&nbsp;</b>2017-04-16<br>
<b>Format description:</b> Polymorphism data from dbSNP<br><!--hTableStart-->
<table bgcolor="#888888" border="0" cellspacing="0" cellpadding="1"><tbody><tr><td>
<table border="1" bgcolor="#FFFEE8" cellspacing="0">
<tbody><tr><th>field</th><th>example</th><th>SQL type</th> <th>description</th> </tr>

<tr><td><tt>bin</tt></td> <td>585</td><td><tt>smallint(5) unsigned</tt></td> <td>Indexing field to speed chromosome range queries.</td></tr>
<tr><td><tt>chrom</tt></td> <td>chr1</td><td><tt>varchar(31)</tt></td> <td>Reference sequence chromosome or scaffold</td></tr>
<tr><td><tt>chromStart</tt></td> <td>10019</td><td><tt>int(10) unsigned</tt></td> <td>Start position in chrom</td></tr>
<tr><td><tt>chromEnd</tt></td> <td>10020</td><td><tt>int(10) unsigned</tt></td> <td>End position in chrom</td></tr>
<tr><td><tt>name</tt></td> <td>rs775809821</td><td><tt>varchar(15)</tt></td> <td>dbSNP Reference SNP (rs) identifier</td></tr>
<tr><td><tt>score</tt></td> <td>0</td><td><tt>smallint(5) unsigned</tt></td> <td>Not used</td></tr>
<tr><td><tt>strand</tt></td> <td>+</td><td><tt>enum('+', '-')</tt></td> <td>Which DNA strand contains the observed alleles</td></tr>
<tr><td><tt>refNCBI</tt></td> <td>A</td><td><tt>blob</tt></td> <td>Reference genomic sequence from dbSNP</td></tr>
<tr><td><tt>refUCSC</tt></td> <td>A</td><td><tt>blob</tt></td> <td>Reference genomic sequence from UCSC lookup of chrom,chromStart,chromEnd</td></tr>
<tr><td><tt>observed</tt></td> <td>-/A</td><td><tt>varchar(255)</tt></td> <td>The sequences of the observed alleles from rs-fasta files</td></tr>
<tr><td><tt>molType</tt></td> <td>genomic</td><td><tt>enum('unknown', 'genomic', 'cDNA', 'mito')</tt></td> <td>Sample type from exemplar submitted SNPs (ss)</td></tr>
<tr><td><tt>class</tt></td> <td>deletion</td><td><tt>enum('single', 'in-del', 'microsatellite', 'named', 'mnp', 'insertion', 'deletion')</tt></td> <td>Class of variant (single, in-del, named, mixed, etc.)</td></tr>
<tr><td><tt>valid</tt></td> <td>unknown</td><td><tt>set('unknown', 'by-cluster', 'by-frequency', 'by-submitter', 'by-2hit-2allele', 'by-hapmap', 'by-1000genomes')</tt></td> <td>Validation status of the SNP</td></tr>
<tr><td><tt>avHet</tt></td> <td>0</td><td><tt>float</tt></td> <td>Average heterozygosity from all observations.  Note: may be computed on small number of samples.</td></tr>
<tr><td><tt>avHetSE</tt></td> <td>0</td><td><tt>float</tt></td> <td>Standard Error for the average heterozygosity</td></tr>
<tr><td><tt>func</tt></td> <td>near-gene-5</td><td><tt>set('unknown', 'coding-synon', 'intron', 'near-gene-3', 'near-gene-5', 'ncRNA', 'nonsense', 'missense', 'stop-loss', 'frameshift', 'cds-indel', 'untranslated-3', 'untranslated-5', 'splice-3', 'splice-5')</tt></td> <td>Functional category of the SNP (coding-synon, coding-nonsynon, intron, etc.)</td></tr>
<tr><td><tt>locType</tt></td> <td>exact</td><td><tt>enum('range', 'exact', 'between', 'rangeInsertion', 'rangeSubstitution', 'rangeDeletion', 'fuzzy')</tt></td> <td>Type of mapping inferred from size on reference; may not agree with class</td></tr>
<tr><td><tt>weight</tt></td> <td>1</td><td><tt>int(10) unsigned</tt></td> <td>The quality of the alignment: 1 = unique mapping, 2 = non-unique, 3 = many matches</td></tr>
<tr><td><tt>exceptions</tt></td> <td>&nbsp;</td><td><tt>set('RefAlleleMismatch', 'DuplicateObserved', 'MixedObserved', 'FlankMismatchGenomeLonger', 'FlankMismatchGenomeEqual', 'FlankMismatchGenomeShorter', 'NamedDeletionZeroSpan', 'NamedInsertionNonzeroSpan', 'SingleClassLongerSpan', 'SingleClassZeroSpan', 'SingleClassTriAllelic', 'SingleClassQuadAllelic', 'ObservedWrongFormat', 'ObservedTooLong', 'ObservedContainsIupac', 'ObservedMismatch', 'MultipleAlignments', 'NonIntegerChromCount', 'AlleleFreqSumNot1', 'SingleAlleleFreq', 'InconsistentAlleles')</tt></td> <td>Unusual conditions noted by UCSC that may indicate a problem with the data</td></tr>
<tr><td><tt>submitterCount</tt></td> <td>1</td><td><tt>smallint(5) unsigned</tt></td> <td>Number of distinct submitter handles for submitted SNPs for this ref SNP</td></tr>
<tr><td><tt>submitters</tt></td> <td>SSMP,</td><td><tt>longblob</tt></td> <td>List of submitter handles</td></tr>
<tr><td><tt>alleleFreqCount</tt></td> <td>0</td><td><tt>smallint(5) unsigned</tt></td> <td>Number of observed alleles with frequency data</td></tr>
<tr><td><tt>alleles</tt></td> <td>&nbsp;</td><td><tt>longblob</tt></td> <td>Observed alleles for which frequency data are available</td></tr>
<tr><td><tt>alleleNs</tt></td> <td>&nbsp;</td><td><tt>longblob</tt></td> <td>Count of chromosomes (2N) on which each allele was observed. Note: this is extrapolated by dbSNP from submitted frequencies and total sample 2N, and is not always an integer.</td></tr>
<tr><td><tt>alleleFreqs</tt></td> <td>&nbsp;</td><td><tt>longblob</tt></td> <td>Allele frequencies</td></tr>
<tr><td><tt>bitfields</tt></td> <td>&nbsp;</td><td><tt>set('unknown', 'clinically-assoc', 'maf-5-some-pop', 'maf-5-all-pops', 'has-omim-omia', 'microattr-tpa', 'submitted-by-lsdb', 'genotype-conflict', 'rs-cluster-nonoverlapping-alleles', 'observed-mismatch')</tt></td> <td>SNP attributes extracted from dbSNP's SNP_bitfield table</td></tr>
</tbody></table>
</td></tr></tbody></table>
<!--hTableEnd-->

	</td><td width="15"></td></tr></tbody></table>
	</td></tr></tbody></table>
	</td></tr></tbody></table>
	
</div>
<div>
<!-- +++++++++++++++++++++ START NEW SECTION +++++++++++++++++++ -->
<br>

   <!--outer table is for border purposes-->
   <table width="100%" bgcolor="#888888" border="0" cellspacing="0" cellpadding="1"><tbody><tr><td>
    <table bgcolor="#FFFEE8" width="100%" border="0" cellspacing="0" cellpadding="0"><tbody><tr><td>

<div class="subheadingBar windowSize">
Connected Tables and Joining Fields     </div>
     <table bgcolor="#FFFEE8" width="100%" cellpadding="0"><tbody><tr><th height="10"></th></tr>
     <tr><td width="10">&nbsp;</td><td>


&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;hg38.<a href="http://genome.ucsc.edu/cgi-bin/hgTables?hgsid=667608919_c94x7S5B0funel50l3ukR6d4tOh5&amp;hgta_doSchemaDb=hg38&amp;hgta_doSchemaTable=snp150Common">snp150Common</a>.exceptions (via snp150.exceptions)<br>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;hg38.<a href="http://genome.ucsc.edu/cgi-bin/hgTables?hgsid=667608919_c94x7S5B0funel50l3ukR6d4tOh5&amp;hgta_doSchemaDb=hg38&amp;hgta_doSchemaTable=snp150ExceptionDesc">snp150ExceptionDesc</a>.exception (via snp150.exceptions)<br>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;hg38.<a href="http://genome.ucsc.edu/cgi-bin/hgTables?hgsid=667608919_c94x7S5B0funel50l3ukR6d4tOh5&amp;hgta_doSchemaDb=hg38&amp;hgta_doSchemaTable=snp150Flagged">snp150Flagged</a>.exceptions (via snp150.exceptions)<br>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;hg38.<a href="http://genome.ucsc.edu/cgi-bin/hgTables?hgsid=667608919_c94x7S5B0funel50l3ukR6d4tOh5&amp;hgta_doSchemaDb=hg38&amp;hgta_doSchemaTable=snp150Mult">snp150Mult</a>.exceptions (via snp150.exceptions)<br>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;hg38.<a href="http://genome.ucsc.edu/cgi-bin/hgTables?hgsid=667608919_c94x7S5B0funel50l3ukR6d4tOh5&amp;hgta_doSchemaDb=hg38&amp;hgta_doSchemaTable=snp150CodingDbSnp">snp150CodingDbSnp</a>.name (via snp150.name)<br>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;hg38.<a href="http://genome.ucsc.edu/cgi-bin/hgTables?hgsid=667608919_c94x7S5B0funel50l3ukR6d4tOh5&amp;hgta_doSchemaDb=hg38&amp;hgta_doSchemaTable=snp150Common">snp150Common</a>.name (via snp150.name)<br>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;hg38.<a href="http://genome.ucsc.edu/cgi-bin/hgTables?hgsid=667608919_c94x7S5B0funel50l3ukR6d4tOh5&amp;hgta_doSchemaDb=hg38&amp;hgta_doSchemaTable=snp150Flagged">snp150Flagged</a>.name (via snp150.name)<br>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;hg38.<a href="http://genome.ucsc.edu/cgi-bin/hgTables?hgsid=667608919_c94x7S5B0funel50l3ukR6d4tOh5&amp;hgta_doSchemaDb=hg38&amp;hgta_doSchemaTable=snp150Mult">snp150Mult</a>.name (via snp150.name)<br>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;hg38.<a href="http://genome.ucsc.edu/cgi-bin/hgTables?hgsid=667608919_c94x7S5B0funel50l3ukR6d4tOh5&amp;hgta_doSchemaDb=hg38&amp;hgta_doSchemaTable=snp150OrthoPt5Pa2Rm8">snp150OrthoPt5Pa2Rm8</a>.name (via snp150.name)<br>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;hg38.<a href="http://genome.ucsc.edu/cgi-bin/hgTables?hgsid=667608919_c94x7S5B0funel50l3ukR6d4tOh5&amp;hgta_doSchemaDb=hg38&amp;hgta_doSchemaTable=snp150Seq">snp150Seq</a>.acc (via snp150.name)<br>

	</td><td width="15"></td></tr></tbody></table>
	</td></tr></tbody></table>
	</td></tr></tbody></table>
	
</div>
<div>
<!-- +++++++++++++++++++++ START NEW SECTION +++++++++++++++++++ -->
<br>

   <!--outer table is for border purposes-->
   <table width="100%" bgcolor="#888888" border="0" cellspacing="0" cellpadding="1"><tbody><tr><td>
    <table bgcolor="#FFFEE8" width="100%" border="0" cellspacing="0" cellpadding="0"><tbody><tr><td>

<div class="subheadingBar windowSize">
Sample Rows     </div>
     <table bgcolor="#FFFEE8" width="100%" cellpadding="0"><tbody><tr><th height="10"></th></tr>
     <tr><td width="10">&nbsp;</td><td>


<!--hTableStart-->
<table bgcolor="#888888" border="0" cellspacing="0" cellpadding="1"><tbody><tr><td>
<table border="1" bgcolor="#FFFEE8" cellspacing="0">
<tbody><tr><th>bin</th><th>chrom</th><th>chromStart</th><th>chromEnd</th><th>name</th><th>score</th><th>strand</th><th>refNCBI</th><th>refUCSC</th><th>observed</th><th>molType</th><th>class</th><th>valid</th><th>avHet</th><th>avHetSE</th><th>func</th><th>locType</th><th>weight</th><th>exceptions</th><th>submitterCount</th><th>submitters</th><th>alleleFreqCount</th><th>alleles</th><th>alleleNs</th><th>alleleFreqs</th><th>bitfields</th></tr><tr><td>585</td><td>chr1</td><td>10019</td><td>10020</td><td>rs775809821</td><td>0</td><td>+</td><td>A</td><td>A</td><td>-/A</td><td>genomic</td><td>deletion</td><td>unknown</td><td>0</td><td>0</td><td>near-gene-5</td><td>exact</td><td>1</td><td></td><td>1</td><td>SSMP,</td><td>0</td><td></td><td></td><td></td><td></td></tr>
<tr><td>585</td><td>chr1</td><td>10038</td><td>10039</td><td>rs978760828</td><td>0</td><td>+</td><td>A</td><td>A</td><td>A/C</td><td>genomic</td><td>single</td><td>unknown</td><td>0</td><td>0</td><td>near-gene-5</td><td>exact</td><td>1</td><td></td><td>1</td><td>USC_VALOUEV,</td><td>0</td><td></td><td></td><td></td><td></td></tr>
<tr><td>585</td><td>chr1</td><td>10042</td><td>10043</td><td>rs1008829651</td><td>0</td><td>+</td><td>T</td><td>T</td><td>A/T</td><td>genomic</td><td>single</td><td>unknown</td><td>0</td><td>0</td><td>near-gene-5</td><td>exact</td><td>1</td><td></td><td>1</td><td>USC_VALOUEV,</td><td>0</td><td></td><td></td><td></td><td></td></tr>
<tr><td>585</td><td>chr1</td><td>10050</td><td>10051</td><td>rs1052373574</td><td>0</td><td>+</td><td>A</td><td>A</td><td>A/G</td><td>genomic</td><td>single</td><td>unknown</td><td>0</td><td>0</td><td>near-gene-5</td><td>exact</td><td>1</td><td></td><td>1</td><td>USC_VALOUEV,</td><td>0</td><td></td><td></td><td></td><td></td></tr>
<tr><td>585</td><td>chr1</td><td>10054</td><td>10055</td><td>rs892501864</td><td>0</td><td>+</td><td>T</td><td>T</td><td>A/T</td><td>genomic</td><td>single</td><td>unknown</td><td>0</td><td>0</td><td>near-gene-5</td><td>exact</td><td>1</td><td></td><td>1</td><td>USC_VALOUEV,</td><td>0</td><td></td><td></td><td></td><td></td></tr>
<tr><td>585</td><td>chr1</td><td>10055</td><td>10055</td><td>rs768019142</td><td>0</td><td>+</td><td>-</td><td>-</td><td>-/A</td><td>genomic</td><td>insertion</td><td>unknown</td><td>0</td><td>0</td><td>near-gene-5</td><td>between</td><td>1</td><td></td><td>1</td><td>SSMP,</td><td>0</td><td></td><td></td><td></td><td></td></tr>
<tr><td>585</td><td>chr1</td><td>10062</td><td>10063</td><td>rs1010989343</td><td>0</td><td>+</td><td>A</td><td>A</td><td>A/C</td><td>genomic</td><td>single</td><td>unknown</td><td>0</td><td>0</td><td>near-gene-5</td><td>exact</td><td>1</td><td></td><td>1</td><td>USC_VALOUEV,</td><td>0</td><td></td><td></td><td></td><td></td></tr>
<tr><td>585</td><td>chr1</td><td>10076</td><td>10077</td><td>rs1022805358</td><td>0</td><td>+</td><td>C</td><td>C</td><td>C/G</td><td>genomic</td><td>single</td><td>unknown</td><td>0</td><td>0</td><td>near-gene-5</td><td>exact</td><td>1</td><td></td><td>1</td><td>USC_VALOUEV,</td><td>0</td><td></td><td></td><td></td><td></td></tr>
<tr><td>585</td><td>chr1</td><td>10107</td><td>10108</td><td>rs62651026</td><td>0</td><td>+</td><td>C</td><td>C</td><td>C/T</td><td>genomic</td><td>single</td><td>unknown</td><td>0</td><td>0</td><td>near-gene-5</td><td>exact</td><td>1</td><td></td><td>1</td><td>BCMHGSC_JDW,</td><td>0</td><td></td><td></td><td></td><td></td></tr>
<tr><td>585</td><td>chr1</td><td>10108</td><td>10109</td><td>rs376007522</td><td>0</td><td>+</td><td>A</td><td>A</td><td>A/T</td><td>genomic</td><td>single</td><td>by-cluster</td><td>0</td><td>0</td><td>near-gene-5</td><td>exact</td><td>1</td><td></td><td>2</td><td>BILGI_BIOE,WEILL_CORNELL_DGM,</td><td>0</td><td></td><td></td><td></td><td></td></tr>
</tbody></table>
</td></tr></tbody></table>
<!--hTableEnd-->
<br><i>Note: all start coordinates in our database are 0-based, not 
1-based.  See explanation 
<a href="http://genome.ucsc.edu/FAQ/FAQtracks#tracks1">here</a>.</i>

	</td><td width="15"></td></tr></tbody></table>
	</td></tr></tbody></table>
	</td></tr></tbody></table>
	
</div>
<div>
<!-- +++++++++++++++++++++ START NEW SECTION +++++++++++++++++++ -->
<br>

   <!--outer table is for border purposes-->
   <table width="100%" bgcolor="#888888" border="0" cellspacing="0" cellpadding="1"><tbody><tr><td>
    <table bgcolor="#FFFEE8" width="100%" border="0" cellspacing="0" cellpadding="0"><tbody><tr><td>

<div class="subheadingBar windowSize">
All SNPs(150) (snp150) Track Description     </div>
     <table bgcolor="#FFFEE8" width="100%" cellpadding="0"><tbody><tr><th height="10"></th></tr>
     <tr><td width="10">&nbsp;</td><td>


<div style="position:relative; top:-1.2em; margin-bottom:0em;"><h2>Description</h2>

<p>
This track contains information about single nucleotide polymorphisms
and small insertions and deletions (indels) — collectively Simple
Nucleotide Polymorphisms — from
<a href="https://www.ncbi.nlm.nih.gov/SNP/" target="_blank">dbSNP</a>
build 150, available from
<a href="ftp://ftp.ncbi.nlm.nih.gov/snp/organisms" target="_blank">ftp.ncbi.nlm.nih.gov/snp</a>.
</p>
<p>
Three tracks contain subsets of the items in this track:
</p><ul>
<li><b>Common SNPs(150):</b> SNPs that have a minor allele frequency
    of at least 1% and are mapped to a single location in the reference
    genome assembly.  Frequency data are not available for all SNPs,
    so this subset is incomplete.</li>
<li><b>Flagged SNPs(150):</b> SNPs flagged as clinically associated by dbSNP,
    mapped to a single location in the reference genome assembly, and
    <em>not</em> known to have a minor allele frequency of at least 1%.
    Frequency data are not available for all SNPs, so this subset may
    include some SNPs whose true minor allele frequency is 1% or greater.</li>
<li><b>Mult. SNPs(150):</b> SNPs that have been mapped to multiple locations
    in the reference genome assembly.  There are very few SNPs in this category
    because dbSNP has been filtering out almost all multiple-mapping SNPs since
    build 149.</li>
</ul>
<p></p>
<p>
The default maximum <a href="http://genome.ucsc.edu/cgi-bin/hgTables#Weight">weight</a> for this track is 1, so unless
the setting is changed in the track controls, SNPs that map to multiple genomic
locations will be omitted from display.  When a SNP's flanking sequences
map to multiple locations in the reference genome, it calls into question
whether there is true variation at those sites, or whether the sequences
at those sites are merely highly similar but not identical.
</p>

The remainder of this page is identical on the following tracks:
<ul>
  <li>Common SNPs(150) - SNPs with &gt;= 1% minor allele frequency (MAF), mapping
     only once to reference assembly.</li>
  <li>Flagged SNPs(150) - SNPs &lt; 1% minor allele frequency (MAF) (or unknown),
     mapping only once to reference assembly,
     flagged in dbSnp as "clinically associated"
     -- <em><b>not necessarily a risk allele!</b></em></li>
  <li>Mult. SNPs(150) - SNPs mapping in more than one place on reference assembly.</li>
  <li>All SNPs(150) - all SNPs from dbSNP mapping to reference assembly.</li>

</ul>

<h2>Interpreting and Configuring the Graphical Display</h2>
<p>
  Variants are shown as single tick marks at most zoom levels.
  When viewing the track at or near base-level resolution, the displayed
  width of the SNP corresponds to the width of the variant in the reference
  sequence. Insertions are indicated by a single tick mark displayed between
  two nucleotides, single nucleotide polymorphisms are displayed as the width
  of a single base, and multiple nucleotide variants are represented by a
  block that spans two or more bases.
</p>

<p>
On the track controls page, SNPs can be colored and/or filtered from the
display according to several attributes:
</p>
  <ul>

    <li>
      <a name="Class"></a>
      <b>Class</b>: Describes the observed alleles<br>
      <ul>
        <li><b>Single</b> - single nucleotide variation: all observed alleles are single nucleotides
	    (can have 2, 3 or 4 alleles)</li>
        <li><b>In-del</b> - insertion/deletion</li>
        <li><b>Heterozygous</b> - heterozygous (undetermined) variation: allele contains string '(heterozygous)'</li>
        <li><b>Microsatellite</b> - the observed allele from dbSNP is a variation in counts of short tandem repeats</li>
        <li><b>Named</b> - the observed allele from dbSNP is given as a text name instead of raw sequence, e.g., (Alu)/-</li>
        <li><b>No Variation</b> - the submission reports an invariant region in the surveyed sequence</li>
        <li><b>Mixed</b> - the cluster contains submissions from multiple classes</li>
        <li><b>Multiple Nucleotide Polymorphism (MNP)</b> - the alleles are all of the same length, and length &gt; 1</li>
        <li><b>Insertion</b> - the polymorphism is an insertion relative to the reference assembly</li>
        <li><b>Deletion</b> - the polymorphism is a deletion relative to the reference assembly</li>
        <li><b>Unknown</b> - no classification provided by data contributor</li>
      </ul>
    </li>


    <li>
      <a name="Valid"></a>
      <b><a href="https://www.ncbi.nlm.nih.gov/SNP/snp_legend.cgi?legend=validation" target="_blank">Validation</a></b>: Method used to validate
	the variant (<i>each variant may be validated by more than one method</i>)<br>
        <ul>
        <li><b>By Frequency</b> - at least one submitted SNP in cluster has frequency data submitted</li>
        <li><b>By Cluster</b> - cluster has at least 2 submissions, with at least one submission assayed with a non-computational method</li>
        <li><b>By Submitter</b> - at least one submitter SNP in cluster was validated by independent assay</li>
        <li><b>By 2 Hit/2 Allele</b> - all alleles have been observed in at least 2 chromosomes</li>
        <li><b>By HapMap</b> (human only) - submitted by <a href="https://hapmap.ncbi.nlm.nih.gov/" target="_blank">HapMap</a> project</li>
        <li><b>By 1000Genomes</b> (human only) - submitted by 
	    <a href="http://www.internationalgenome.org/" target="_blank">1000Genomes</a> project</li>
        <li><b>Unknown</b> - no validation has been reported for this variant</li>
      </ul>
    </li>
    <li>
      <a name="Func"></a>
      <b>Function</b>: dbSNP's predicted functional effect of variant on RefSeq transcripts,
      both curated (NM_* and NR_*) as in the RefSeq Genes track and predicted (XM_* and XR_*),
      not shown in UCSC Genome Browser.
      A variant may have more than one functional role if it overlaps
      multiple transcripts.
      These terms and definitions are from the <a href="http://www.sequenceontology.org/" target="_BLANK">Sequence Ontology (SO)</a>; click on a term to view it in the
      <a href="http://www.sequenceontology.org/browser/browser/obob.cgi?release=current_release" target="_BLANK">MISO Sequence Ontology Browser</a>.<br>
      <ul>
        <li><b>Unknown</b> - no functional classification provided (possibly intergenic)</li>
        <li><b><a href="http://www.sequenceontology.org/browser/current_release/term/SO:0001819" target="_BLANK">synonymous_variant</a></b> -
	    A sequence variant where there is no resulting change to the encoded amino acid
	    (dbSNP term: <tt>coding-synon</tt>)</li>
        <li><b><a href="http://www.sequenceontology.org/browser/current_release/term/SO:0001627" target="_BLANK">intron_variant</a></b> -
	    A transcript variant occurring within an intron
	    (dbSNP term: <tt>intron</tt>)</li>
        <li><b><a href="http://www.sequenceontology.org/browser/current_release/term/SO:0001632" target="_BLANK">downstream_gene_variant</a></b> -
	    A sequence variant located 3' of a gene
	    (dbSNP term: <tt>near-gene-3</tt>)</li>
        <li><b><a href="http://www.sequenceontology.org/browser/current_release/term/SO:0001631" target="_BLANK">upstream_gene_variant</a></b> -
	    A sequence variant located 5' of a gene
	    (dbSNP term: <tt>near-gene-5</tt>)</li>
        <li><b><a href="http://www.sequenceontology.org/browser/current_release/term/SO:0001619" target="_BLANK">nc_transcript_variant</a></b> -
	    A transcript variant of a non coding RNA gene
	    (dbSNP term: <tt>ncRNA</tt>)</li>
	 <li><b><a href="http://www.sequenceontology.org/browser/current_release/term/SO:0001587" target="_BLANK">stop_gained</a></b> -
	    A sequence variant whereby at least one base of a codon is changed, resulting in
	    a premature stop codon, leading to a shortened transcript
	    (dbSNP term: <tt>nonsense</tt>)</li>
        <li><b><a href="http://www.sequenceontology.org/browser/current_release/term/SO:0001583" target="_BLANK">missense_variant</a></b> -
	    A sequence variant, where the change may be longer than 3 bases, and at least
	    one base of a codon is changed resulting in a codon that encodes for a
	    different amino acid
	    (dbSNP term: <tt>missense</tt>)</li>
        <li><b><a href="http://www.sequenceontology.org/browser/current_release/term/SO:0001578" target="_BLANK">stop_lost</a></b> -
	    A sequence variant where at least one base of the terminator codon (stop)
	    is changed, resulting in an elongated transcript
	    (dbSNP term: <tt>stop-loss</tt>)</li>
        <li><b><a href="http://www.sequenceontology.org/browser/current_release/term/SO:0001589" target="_BLANK">frameshift_variant</a></b> -
	    A sequence variant which causes a disruption of the translational reading frame,
	    because the number of nucleotides inserted or deleted is not a multiple of three
	    (dbSNP term: <tt>frameshift</tt>)</li>
        <li><b><a href="http://www.sequenceontology.org/browser/current_release/term/SO:0001820" target="_BLANK">inframe_indel</a></b> -
	    A coding sequence variant where the change does not alter the frame
	    of the transcript
	    (dbSNP term: <tt>cds-indel</tt>)</li>
        <li><b><a href="http://www.sequenceontology.org/browser/current_release/term/SO:0001624" target="_BLANK">3_prime_UTR_variant</a></b> -
	    A UTR variant of the 3' UTR
	    (dbSNP term: <tt>untranslated-3</tt>)</li>
        <li><b><a href="http://www.sequenceontology.org/browser/current_release/term/SO:0001623" target="_BLANK">5_prime_UTR_variant</a></b> -
	    A UTR variant of the 5' UTR
	    (dbSNP term: <tt>untranslated-5</tt>)</li>
        <li><b><a href="http://www.sequenceontology.org/browser/current_release/term/SO:0001574" target="_BLANK">splice_acceptor_variant</a></b> -
	    A splice variant that changes the 2 base region at the 3' end of an intron
	    (dbSNP term: <tt>splice-3</tt>)</li>
        <li><b><a href="http://www.sequenceontology.org/browser/current_release/term/SO:0001575" target="_BLANK">splice_donor_variant</a></b> -
	    A splice variant that changes the 2 base region at the 5' end of an intron
	    (dbSNP term: <tt>splice-5</tt>)</li>
      </ul>
      In the Coloring Options section of the track controls page,
      function terms are grouped into several categories, shown here with default colors.
      If a SNP has more than one of these attributes, the stronger color will override 
      the weaker color. The order of colors, from strongest to weakest, is red, green, 
      blue, gray, and black.
      <ul>
	<li><span style="color: #000000; font-weight: bold;">Locus:</span>
	    <a href="http://www.sequenceontology.org/browser/current_release/term/SO:0001632" target="_BLANK">downstream_gene_variant</a>,
	    <a href="http://www.sequenceontology.org/browser/current_release/term/SO:0001631" target="_BLANK">upstream_gene_variant</a></li>
	<li><span style="color: #00ff00; font-weight: bold;">Coding - Synonymous:</span>
	    <a href="http://www.sequenceontology.org/browser/current_release/term/SO:0001819" target="_BLANK">synonymous_variant</a>
	</li><li><span style="color: #ff0000; font-weight: bold;">Coding - Non-Synonymous:</span>
	    <a href="http://www.sequenceontology.org/browser/current_release/term/SO:0001587" target="_BLANK">stop_gained</a>,
	    <a href="http://www.sequenceontology.org/browser/current_release/term/SO:0001583" target="_BLANK">missense_variant</a>,
	    <a href="http://www.sequenceontology.org/browser/current_release/term/SO:0001578" target="_BLANK">stop_lost</a>,
	    <a href="http://www.sequenceontology.org/browser/current_release/term/SO:0001589" target="_BLANK">frameshift_variant</a>,
	    <a href="http://www.sequenceontology.org/browser/current_release/term/SO:0001820" target="_BLANK">inframe_indel</a></li>
        <li><span style="color: #0000ff; font-weight: bold;">Untranslated:</span>
	    <a href="http://www.sequenceontology.org/browser/current_release/term/SO:0001623" target="_BLANK">5_prime_UTR_variant</a>,
	    <a href="http://www.sequenceontology.org/browser/current_release/term/SO:0001624" target="_BLANK">3_prime_UTR_variant</a></li>
	<li><span style="color: #000000; font-weight: bold;">Intron:</span>
	    <a href="http://www.sequenceontology.org/browser/current_release/term/SO:0001627" target="_BLANK">intron_variant</a></li>
	<li><span style="color: #ff0000; font-weight: bold;">Splice Site:</span>
	    <a href="http://www.sequenceontology.org/browser/current_release/term/SO:0001574" target="_BLANK">splice_acceptor_variant</a>,
	    <a href="http://www.sequenceontology.org/browser/current_release/term/SO:0001575" target="_BLANK">splice_donor_variant</a>
      </li></ul>
      <span style="color: #0000ff; font-weight: bold;">Non-coding (ncRNA):</span>
      (<a href="http://www.sequenceontology.org/browser/current_release/term/SO:0001619" target="_BLANK">nc_transcript_variant</a>) are colored blue.
    </li>
    <li>
      <a name="MolType"></a>
      <b>Molecule Type</b>: Sample used to find this variant<br>
      <ul>
        <li><b>Genomic</b> - variant discovered using a genomic template</li>
        <li><b>cDNA</b> - variant discovered using a cDNA template</li>
        <li><b>Unknown</b> - sample type not known</li>
      </ul>
    </li>
    <li>
      <a name="Exceptions"></a>
      <b>Unusual Conditions (UCSC)</b>: UCSC checks for several anomalies
      that may indicate a problem with the mapping, and reports them in the
      Annotations section of the SNP details page if found:
      <ul>
        <li><b>AlleleFreqSumNot1</b> - Allele frequencies do not sum
            to 1.0 (+-0.01).  This SNP's allele frequency data are
	    probably incomplete.</li>
        <li><b>DuplicateObserved</b>,
            <b>MixedObserved</b> - Multiple distinct insertion SNPs have
	    been mapped to this location, with either the same inserted
	    sequence (Duplicate) or different inserted sequence (Mixed).</li>
        <li><b>FlankMismatchGenomeEqual</b>,
	    <b>FlankMismatchGenomeLonger</b>,
	    <b>FlankMismatchGenomeShorter</b> - NCBI's alignment of
            the flanking sequences had at least one mismatch or gap
	    near the mapped SNP position.
            (UCSC's re-alignment of flanking sequences to the genome may
            be informative.)</li>
        <li><b>MultipleAlignments</b> - This SNP's flanking sequences
            align to more than one location in the reference assembly.</li>
        <li><b>NamedDeletionZeroSpan</b> - A deletion (from the
            genome) was observed but the annotation spans 0 bases.
            (UCSC's re-alignment of flanking sequences to the genome may
            be informative.)</li>
        <li><b>NamedInsertionNonzeroSpan</b> - An insertion (into the
            genome) was observed but the annotation spans more than 0
            bases.  (UCSC's re-alignment of flanking sequences to the
            genome may be informative.)</li>
        <li><b>NonIntegerChromCount</b> - At least one allele
            frequency corresponds to a non-integer (+-0.010000) count of
            chromosomes on which the allele was observed.  The reported
            total sample count for this SNP is probably incorrect.</li>
        <li><b>ObservedContainsIupac</b> - At least one observed allele
            from dbSNP contains an <a href="http://genome.ucsc.edu/goldenPath/help/iupac.html" target="_blank">IUPAC</a> ambiguous base (e.g., R, Y, N).</li>
        <li><b>ObservedMismatch</b> - UCSC reference allele does not
            match any observed allele from dbSNP.  This is tested only
	    for SNPs whose class is single, in-del, insertion, deletion,
	    mnp or mixed.</li>
        <li><b>ObservedTooLong</b> - Observed allele not given (length
            too long).</li>
        <li><b>ObservedWrongFormat</b> - Observed allele(s) from dbSNP
            have unexpected format for the given class.</li>
        <li><b>RefAlleleMismatch</b> - The reference allele from dbSNP
            does not match the UCSC reference allele, i.e., the bases in
	    the mapped position range.</li>
        <li><b>RefAlleleRevComp</b> - The reference allele from dbSNP
            matches the reverse complement of the UCSC reference
            allele.</li>
        <li><b>SingleClassLongerSpan</b> - All observed alleles are
            single-base, but the annotation spans more than 1 base.
            (UCSC's re-alignment of flanking sequences to the genome may
            be informative.)</li>
        <li><b>SingleClassZeroSpan</b> - All observed alleles are
            single-base, but the annotation spans 0 bases.  (UCSC's
            re-alignment of flanking sequences to the genome may be
            informative.)</li>
      </ul>
      Another condition, which does not necessarily imply any problem,
      is noted:
      <ul>
        <li><b>SingleClassTriAllelic</b>, <b>SingleClassQuadAllelic</b> -
            Class is single and three or four different bases have been
	    observed (usually there are only two).</li>
      </ul>
    </li>
    <li>
      <a name="Bitfields"></a>
      <b>Miscellaneous Attributes (dbSNP)</b>: several properties extracted
         from dbSNP's SNP_bitfield table
         (see <a href="ftp://ftp.ncbi.nlm.nih.gov/snp/specs/dbSNP_BitField_v5.pdf" target="_blank">dbSNP_BitField_v5.pdf</a> for details)
      <ul>
        <li><b>Clinically Associated</b> (human only) - SNP is in OMIM and/or at
	    least one submitter is a Locus-Specific Database.  This does
	    not necessarily imply that the variant causes any disease,
	    only that it has been observed in clinical studies.</li>
        <li><b>Appears in OMIM/OMIA</b> - SNP is mentioned in
	    <a href="https://www.ncbi.nlm.nih.gov/omim" target="_blank">Online Mendelian Inheritance in Man</a> for
	    human SNPs, or <a href="http://omia.org/home/" target="_blank">Online Mendelian Inheritance in Animals</a> for
	    non-human animal SNPs.  Some of these SNPs are quite common,
	    others are known to cause disease; see OMIM/OMIA for more
	    information.</li>
        <li><b>Has Microattribution/Third-Party Annotation</b> - At least
	    one of the SNP's submitters studied this SNP in a biomedical
	    setting, but is not a Locus-Specific Database or OMIM/OMIA.</li>
        <li><b>Submitted by Locus-Specific Database</b> - At least one of
	    the SNP's submitters is associated with a database of variants
	    associated with a particular gene.  These variants may or may
	    not be known to be causative.</li>
        <li><b>MAF &gt;= 5% in Some Population</b> - Minor Allele Frequency is
	    at least 5% in at least one population assayed.</li>
        <li><b>MAF &gt;= 5% in All Populations</b> - Minor Allele Frequency is
	    at least 5% in all populations assayed.</li>
        <li><b>Genotype Conflict</b> - Quality check: different genotypes
	    have been submitted for the same individual.</li>
        <li><b>Ref SNP Cluster has Non-overlapping Alleles</b> - Quality
	    check: this reference SNP was clustered from submitted SNPs
	    with non-overlapping sets of observed alleles.</li>
        <li><b>Some Assembly's Allele Does Not Match Observed</b> -
	    Quality check: at least one assembly mapped by dbSNP has an allele
            at the mapped position that is not present in this SNP's observed
            alleles.</li>
      </ul>
    </li>
  </ul>
  Several other properties do not have coloring options, but do have
  some filtering options:
  <ul>
    <li>
      <a name="AvHet"></a>
      <b>Average heterozygosity</b>: Calculated by dbSNP as described in
      <a href="https://www.ncbi.nlm.nih.gov/SNP/Hetfreq.html" target="_blank">Computation of Average
      Heterozygosity and Standard Error for dbSNP RefSNP Clusters</a>.
      <ul>
      <li> Average heterozygosity should not exceed 0.5 for bi-allelic
           single-base substitutions.</li>
      </ul>
    </li>
     <li>
      <a name="Weight"></a>
      <b>Weight</b>: Alignment quality assigned by dbSNP. Before dbSNP build
        147, weight had values 1, 2 or 3, with 1 being the highest quality
        (mapped to a single genomic location). As of dbSNP build 147, dbSNP
        now releases only the variants with weight 1.
    </li>
     <li>
      <a name="Submitters"></a>
      <b>Submitter handles</b>: These are short, single-word identifiers of
      labs or consortia that submitted SNPs that were clustered into this
      reference SNP by dbSNP (e.g., 1000GENOMES, ENSEMBL, KWOK).  Some SNPs
      have been observed by many different submitters, and some by only a
      single submitter (although that single submitter may have tested a
      large number of samples).
    </li>
    <li>
      <a name="AlleleFreq"></a>
      <b>AlleleFrequencies</b>: Some submissions to dbSNP include
      allele frequencies and the study's sample size
      (i.e., the number of distinct chromosomes, which is two times the
      number of individuals assayed, a.k.a. 2N).  dbSNP combines all
      available frequencies and counts from submitted SNPs that are
      clustered together into a reference SNP.
    </li>
  </ul>

 <p>
 You can configure this track such that the details page displays
 the function and coding differences relative to
 particular gene sets.  Choose the gene sets from the list on the SNP
 configuration page displayed beneath this heading: <em>On details page,
 show function and coding differences relative to</em>.
 When one or more gene tracks are selected, the SNP details page
 lists all genes that the SNP hits (or is close to), with the same keywords
 used in the <a href="http://genome.ucsc.edu/cgi-bin/hgTables#Func">function</a> category.  The function usually
 agrees with NCBI's function, except when NCBI's functional annotation is
 relative to an XM_* predicted RefSeq (not included in the UCSC Genome
 Browser's RefSeq Genes track) and/or UCSC's functional annotation is
 relative to a transcript that is not in RefSeq.
 </p>

<h2>Insertions/Deletions</h2>
<p>
dbSNP uses a class called 'in-del'.  We compare the length of the
reference allele to the length(s) of observed alleles; if the
reference allele is shorter than all other observed alleles, we change
'in-del' to 'insertion'.  Likewise, if the reference allele is longer
than all other observed alleles, we change 'in-del' to 'deletion'.
</p>

<h2>UCSC Re-alignment of flanking sequences</h2>
<p>
dbSNP determines the genomic locations of SNPs by aligning their flanking
sequences to the genome.
UCSC displays SNPs in the locations determined by dbSNP, but does not
have access to the alignments on which dbSNP based its mappings.
Instead, UCSC re-aligns the flanking sequences
to the neighboring genomic sequence for display on SNP details pages.
While the recomputed alignments may differ from dbSNP's alignments,
they often are informative when UCSC has annotated an unusual condition.
</p>
<p>
Non-repetitive genomic sequence is shown in upper case like the flanking
sequence, and a "|" indicates each match between genomic and flanking bases.
Repetitive genomic sequence (annotated by RepeatMasker and/or the
Tandem Repeats Finder with period &gt;= 12) is shown in lower case, and matching
bases are indicated by a "+".
</p>

<h2>Data Sources and Methods</h2>

<p>
The data that comprise this track were extracted from database dump files
and headers of fasta files downloaded from NCBI.
The database dump files were downloaded from
<a href="ftp://ftp.ncbi.nlm.nih.gov/snp/organisms/human_9606_b150_GRCh37p13/database/data/organism_data/" target="_BLANK">ftp://ftp.ncbi.nlm.nih.gov/snp/organisms/human_9606_b150_GRCh37p13/database/data/organism_data/</a>
for hg19 and from
<a href="ftp://ftp.ncbi.nlm.nih.gov/snp/organisms/human_9606_b150_GRCh38p7/database/data/organism_data/" target="_BLANK">ftp://ftp.ncbi.nlm.nih.gov/snp/organisms/human_9606_b150_GRCh38p7/database/data/organism_data/</a>
for hg38.
The fasta files were downloaded from
<a href="ftp://ftp.ncbi.nlm.nih.gov/snp/organisms/human_9606_b150_GRCh37p13/rs_fasta/" target="_BLANK">ftp://ftp.ncbi.nlm.nih.gov/snp/organisms/human_9606_b150_GRCh37p13/rs_fasta/</a>
for hg19 and from
<a href="ftp://ftp.ncbi.nlm.nih.gov/snp/organisms/human_9606_b150_GRCh38p7/rs_fasta/" target="_BLANK">ftp://ftp.ncbi.nlm.nih.gov/snp/organisms/human_9606_b150_GRCh38p7/rs_fasta/</a>
for hg38.
</p>
  <ul>
  <li>Coordinates, orientation, location type and dbSNP reference allele data
      were obtained from b150_SNPContigLoc_<em>N</em>.bcp.gz and
    b150_ContigInfo_<em>N</em>.bcp.gz.  (<em>N</em> = 105 for hg19, 107 for hg38)</li>
  <li>b150_SNPMapInfo_<em>N</em>.bcp.gz provided the alignment weights.
  </li><li>Functional classification was obtained from
      b150_SNPContigLocusId_<em>N</em>.bcp.gz. The internal database representation
      uses dbSNP's function terms, but for display in SNP details pages,
      these are translated into
      <a href="http://www.sequenceontology.org/" target="_BLANK">Sequence Ontology</a> terms.</li>
  <li>Validation status and heterozygosity were obtained from SNP.bcp.gz.</li>
  <li>SNPAlleleFreq.bcp.gz and ../shared/Allele.bcp.gz provided allele frequencies.
      For the human assembly, allele frequencies were also taken from
      SNPAlleleFreq_TGP.bcp.gz .</li>
  <li>Submitter handles were extracted from Batch.bcp.gz, SubSNP.bcp.gz and
      SNPSubSNPLink.bcp.gz.</li>
  <li>SNP_bitfield.bcp.gz provided miscellaneous properties annotated by dbSNP,
       such as clinically-associated.  See the document
       <a href="ftp://ftp.ncbi.nlm.nih.gov/snp/specs/dbSNP_BitField_v5.pdf" target="_blank">dbSNP_BitField_v5.pdf</a> for details.</li>
  <li>The header lines in the rs_fasta files were used for molecule type,
      class and observed polymorphism.</li>
  </ul>

<h2>Data Access</h2>
<p>
The raw data can be explored interactively with the <a href="http://genome.ucsc.edu/cgi-bin/hgTables" target="_blank">Table Browser</a>,
<a href="http://genome.ucsc.edu/cgi-bin/hgIntegrator" target="_blank">Data Integrator</a>, or <a href="http://genome.ucsc.edu/cgi-bin/hgVai" target="_blank">Variant Annotation Integrator</a>.
For automated analysis, the genome annotation can be downloaded from the downloads server for <a href="http://hgdownload.soe.ucsc.edu/goldenPath/hg38/database/" target="_blank"> hg38</a> and
<a href="http://hgdownload.soe.ucsc.edu/goldenPath/hg19/database/" target="_blank"> hg19</a> (snp150*.txt.gz) or the <a href="http://genome.ucsc.edu/goldenPath/help/mysql.html" target="_blank">public MySQL server</a>.
Please refer to our <a href="https://groups.google.com/a/soe.ucsc.edu/forum/?hl=en&amp;fromgroups#!search/download+snps" target="_blank">mailing list archives</a>
for questions and example queries, or our <a href="http://genome.ucsc.edu/FAQ/FAQdownloads.html#download36" target="_blank">Data Access FAQ</a> for more information.
</p>


<h2>Orthologous Alleles (human assemblies only)</h2>
<p>
For the human assembly, we provide a related table that contains
orthologous alleles in the chimpanzee, orangutan and rhesus macaque
reference genome assemblies.
We use our liftOver utility to identify the orthologous alleles.
The candidate human SNPs are a filtered list that meet the criteria:
</p><ul>
<li>class = 'single'</li>
<li>mapped position in the human reference genome is one base long</li>
<li>aligned to only one location in the human reference genome</li>
<li>not aligned to a chrN_random chrom</li>
<li>biallelic (not tri- or quad-allelic)</li>
</ul>

In some cases the orthologous allele is unknown; these are set to 'N'.
If a lift was not possible, we set the orthologous allele to '?' and the
orthologous start and end position to 0 (zero).

<h2>Masked FASTA Files (human assemblies only)</h2>

FASTA files that have been modified to use
<a href="http://genome.ucsc.edu/goldenPath/help/iupac.html">IUPAC</a>
ambiguous nucleotide characters at
each base covered by a single-base substitution are available for download:
<a href="http://hgdownload.soe.ucsc.edu/goldenPath/hg19/snp150Mask/">GRCh37/hg19</a>,
<a href="http://hgdownload.soe.ucsc.edu/goldenPath/hg38/snp150Mask/">GRCh38/hg38</a>.
Note that only single-base substitutions (no insertions or deletions) were used
to mask the sequence, and these were filtered to exclude problematic SNPs.

<h2>References</h2>
<p>
Sherry ST, Ward MH, Kholodov M, Baker J, Phan L, Smigielski EM, Sirotkin K.
<a href="https://academic.oup.com/nar/article/29/1/308/1116004/dbSNP-the-NCBI-database-of-genetic-variation" target="_blank">dbSNP: the NCBI database of genetic variation</a>.
<em>Nucleic Acids Res</em>. 2001 Jan 1;29(1):308-11.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/11125122" target="_blank">11125122</a>; PMC: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC29783/" target="_blank">PMC29783</a>
</p>


</div>
	</td><td width="15"></td></tr></tbody></table>
	</td></tr></tbody></table>
	</td></tr></tbody></table>
	
</div>
</td></tr></tbody></table>

<script type="text/javascript" nonce="7XrxAeT7uqKhKVCaESUu8jIIOF00">
function showWarnBox() {document.getElementById('warnOK').innerHTML='&nbsp;OK&nbsp;';var warnBox=document.getElementById('warnBox');warnBox.style.display=''; warnBox.style.width='65%';document.getElementById('warnHead').innerHTML='Warning/Error(s):';window.scrollTo(0, 0);}
function hideWarnBox() {var warnBox=document.getElementById('warnBox');warnBox.style.display='none';var warnList=document.getElementById('warnList');warnList.innerHTML='';var endOfPage = document.body.innerHTML.substr(document.body.innerHTML.length-20);if(endOfPage.lastIndexOf('-- ERROR --') > 0) { history.back(); }}
document.getElementById('warnOK').onclick = function() {hideWarnBox();return false;};
window.onunload = function(){}; // Trick to avoid FF back button issue.
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
  ga('create', 'UA-4289047-4', 'auto');
  ga('require', 'displayfeatures');
  ga('send', 'pageview');

</script>

<span style="border-radius: 3px; text-indent: 20px; width: auto; padding: 0px 4px 0px 0px; text-align: center; font-style: normal; font-variant: normal; font-weight: bold; font-stretch: normal; font-size: 11px; line-height: 20px; font-family: &quot;Helvetica Neue&quot;, Helvetica, sans-serif; color: rgb(255, 255, 255); background: url(&quot;data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMzBweCIgd2lkdGg9IjMwcHgiIHZpZXdCb3g9Ii0xIC0xIDMxIDMxIj48Zz48cGF0aCBkPSJNMjkuNDQ5LDE0LjY2MiBDMjkuNDQ5LDIyLjcyMiAyMi44NjgsMjkuMjU2IDE0Ljc1LDI5LjI1NiBDNi42MzIsMjkuMjU2IDAuMDUxLDIyLjcyMiAwLjA1MSwxNC42NjIgQzAuMDUxLDYuNjAxIDYuNjMyLDAuMDY3IDE0Ljc1LDAuMDY3IEMyMi44NjgsMC4wNjcgMjkuNDQ5LDYuNjAxIDI5LjQ0OSwxNC42NjIiIGZpbGw9IiNmZmYiIHN0cm9rZT0iI2ZmZiIgc3Ryb2tlLXdpZHRoPSIxIj48L3BhdGg+PHBhdGggZD0iTTE0LjczMywxLjY4NiBDNy41MTYsMS42ODYgMS42NjUsNy40OTUgMS42NjUsMTQuNjYyIEMxLjY2NSwyMC4xNTkgNS4xMDksMjQuODU0IDkuOTcsMjYuNzQ0IEM5Ljg1NiwyNS43MTggOS43NTMsMjQuMTQzIDEwLjAxNiwyMy4wMjIgQzEwLjI1MywyMi4wMSAxMS41NDgsMTYuNTcyIDExLjU0OCwxNi41NzIgQzExLjU0OCwxNi41NzIgMTEuMTU3LDE1Ljc5NSAxMS4xNTcsMTQuNjQ2IEMxMS4xNTcsMTIuODQyIDEyLjIxMSwxMS40OTUgMTMuNTIyLDExLjQ5NSBDMTQuNjM3LDExLjQ5NSAxNS4xNzUsMTIuMzI2IDE1LjE3NSwxMy4zMjMgQzE1LjE3NSwxNC40MzYgMTQuNDYyLDE2LjEgMTQuMDkzLDE3LjY0MyBDMTMuNzg1LDE4LjkzNSAxNC43NDUsMTkuOTg4IDE2LjAyOCwxOS45ODggQzE4LjM1MSwxOS45ODggMjAuMTM2LDE3LjU1NiAyMC4xMzYsMTQuMDQ2IEMyMC4xMzYsMTAuOTM5IDE3Ljg4OCw4Ljc2NyAxNC42NzgsOC43NjcgQzEwLjk1OSw4Ljc2NyA4Ljc3NywxMS41MzYgOC43NzcsMTQuMzk4IEM4Ljc3NywxNS41MTMgOS4yMSwxNi43MDkgOS43NDksMTcuMzU5IEM5Ljg1NiwxNy40ODggOS44NzIsMTcuNiA5Ljg0LDE3LjczMSBDOS43NDEsMTguMTQxIDkuNTIsMTkuMDIzIDkuNDc3LDE5LjIwMyBDOS40MiwxOS40NCA5LjI4OCwxOS40OTEgOS4wNCwxOS4zNzYgQzcuNDA4LDE4LjYyMiA2LjM4NywxNi4yNTIgNi4zODcsMTQuMzQ5IEM2LjM4NywxMC4yNTYgOS4zODMsNi40OTcgMTUuMDIyLDYuNDk3IEMxOS41NTUsNi40OTcgMjMuMDc4LDkuNzA1IDIzLjA3OCwxMy45OTEgQzIzLjA3OCwxOC40NjMgMjAuMjM5LDIyLjA2MiAxNi4yOTcsMjIuMDYyIEMxNC45NzMsMjIuMDYyIDEzLjcyOCwyMS4zNzkgMTMuMzAyLDIwLjU3MiBDMTMuMzAyLDIwLjU3MiAxMi42NDcsMjMuMDUgMTIuNDg4LDIzLjY1NyBDMTIuMTkzLDI0Ljc4NCAxMS4zOTYsMjYuMTk2IDEwLjg2MywyNy4wNTggQzEyLjA4NiwyNy40MzQgMTMuMzg2LDI3LjYzNyAxNC43MzMsMjcuNjM3IEMyMS45NSwyNy42MzcgMjcuODAxLDIxLjgyOCAyNy44MDEsMTQuNjYyIEMyNy44MDEsNy40OTUgMjEuOTUsMS42ODYgMTQuNzMzLDEuNjg2IiBmaWxsPSIjYmQwODFjIj48L3BhdGg+PC9nPjwvc3ZnPg==&quot;) 3px 50% / 14px 14px no-repeat rgb(189, 8, 28); position: absolute; opacity: 1; z-index: 8675309; display: none; cursor: pointer; border: none; -webkit-font-smoothing: antialiased;">Save</span><span style="width: 24px; height: 24px; background: url(&quot;data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiA/Pjxzdmcgd2lkdGg9IjI0cHgiIGhlaWdodD0iMjRweCIgdmlld0JveD0iMCAwIDI0IDI0IiB2ZXJzaW9uPSIxLjEiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgeG1sbnM6eGxpbms9Imh0dHA6Ly93d3cudzMub3JnLzE5OTkveGxpbmsiPjxkZWZzPjxtYXNrIGlkPSJtIj48cmVjdCBmaWxsPSIjZmZmIiB4PSIwIiB5PSIwIiB3aWR0aD0iMjQiIGhlaWdodD0iMjQiIHJ4PSI2IiByeT0iNiIvPjxyZWN0IGZpbGw9IiMwMDAiIHg9IjUiIHk9IjUiIHdpZHRoPSIxNCIgaGVpZ2h0PSIxNCIgcng9IjEiIHJ5PSIxIi8+PHJlY3QgZmlsbD0iIzAwMCIgeD0iMTAiIHk9IjAiIHdpZHRoPSI0IiBoZWlnaHQ9IjI0Ii8+PHJlY3QgZmlsbD0iIzAwMCIgeD0iMCIgeT0iMTAiIHdpZHRoPSIyNCIgaGVpZ2h0PSI0Ii8+PC9tYXNrPjwvZGVmcz48cmVjdCB4PSIwIiB5PSIwIiB3aWR0aD0iMjQiIGhlaWdodD0iMjQiIGZpbGw9IiNmZmYiIG1hc2s9InVybCgjbSkiLz48L3N2Zz4=&quot;) 50% 50% / 14px 14px no-repeat rgba(0, 0, 0, 0.4); position: absolute; opacity: 1; z-index: 8675309; display: none; cursor: pointer; border: none; border-radius: 12px;"></span></body></html>